src/testIntegration/java/uk/gov/justice/digital/delius/controller/secure/OffendersResource_getOffenderContactSummariesByCrn.java

```diff
@@ -0,0 +1,137 @@
+package uk.gov.justice.digital.delius.controller.secure;
+
+import io.restassured.specification.Argument;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import java.util.List;
+
+import static io.restassured.RestAssured.given;
+import static io.restassured.RestAssured.withArgs;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.greaterThan;
+import static org.hamcrest.Matchers.notNullValue;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
```

Comment:
very minor, you could omit SpringBootTest it is one the base class

Reply:
TBH, Im not actually sure what the issue with LocalDateTime is - I suppose the UTC -> GMT would have to be done in the client if we didn't do it here. At this point I've only used it to remain consistent with the existing appointment APIs created by interventions team. What are your thoughts on it?